data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3482 @@
+			choice
+				`	(It is just a light flicking, no need to worry.)`
+				`	"GET DOWN!"`
+					goto down
+			`	It was indeed just a light flicking. From a laser sight, used to acquire a target: you.`
+				die
+			
+			label down
+			`	You get down yourself as you give the warning, then look his way to see if he followed it. He was evidently trained for this, as you shout he does not hesitate nor does he ask any questions. He promptly jumps to the side and, once in a stable position on the ground, fires his rifle. You are impressed by the precision of that weapon as the sniper was quite clearly taken out with the first shot, but still your armored companion fires three more times.`
+			`	After searching through the vault to find other surprises, the area is declared cleared. Any other stalkers have probably run off after seeing the firepower the Hai brought. You then run the necessary tests on the jump drives whilst he takes care of the security systems.`
+			choice
+				`	(Go back to the ship.)`
+					goto ship
+				`	"We could have just come into the vault swarming with ground forces instead of risking our lives..."`
+				`	"Why did we have to go into the vault instead of just connecting it to the ship earlier?"`
+					goto vault
+			`	"As you have probably noticed, we did not have access to this vault for quite some time. Some unfettered humans could have snuck in and sabotaged the jump drives. To cripple your ship, or worse." As you head back to your ship, you notice more activity than on the way in. He keeps his rifle well in sight in order to dissuade any potential attackers.`
+				goto ship
+			
+			label vault
+			`	"The point of all of this was to be discrete; jump drives are worth a lot on the black market. We also needed to make sure there was not any malfunctions with them. Even monkeys could rig them to blow when placed on a ship and used to jump. Besides, we want to keep using this vault for later, so we may as well ensure its safety."`
+			choice
+				`	(Go back to the ship.)`
+					goto ship
+				`	"If this world is so wild, why put the jump drives here in the first place?"`
+			`	"The Unfettered are not the ones who placed those jump drives here..." He really does not seem to want to tell you much about this vault.`
+			
+			label ship
+			`	You go back to your ship with the Hai, and give the orders to collect and install the jump drives.`
+			`	A bit later the Hai comes to speak to you on the bridge. "It took us too long to install the jump drives, and we seem to have attracted attention. I guess that if some of the unfettered humans got away from the vault, they have contacted some friends..." As you look outside you see multiple ships getting ready to launch far away. You launch as soon as you can, rushing trough the correct procedures.`
```

Comment:
Logically you'd be more likely to detect that other ships had launched on your scanners, not actually witness anything useful on this front with your eyeballs.

Reply:
I'll write something about alarm going off to notify you of enemy ships launching